img/Project 3.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<!-- saved from url=(0037)http://127.0.0.1:5500/code/index.html -->
+<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Project 3</title>
+  
+  
+<link rel="stylesheet" href="./Project 3_files/mobile-style.css"><link rel="stylesheet" href="./Project 3_files/overlay.css"></head>
+
+<body data-new-gr-c-s-check-loaded="14.993.0" data-gr-ext-installed="">
+
+  <header>
+    <!-- Navigation Bar (Menu /Search /ShoppingCar /Login ) :) -->
+    <section class="nav-bar">
+      <!-- Navigation Bar (Menu  -->
+      <div class="menu-hamburguesa">
+        <a class="button" href="http://127.0.0.1:5500/code/index.html#modal">
+          <img src="./Project 3_files/icon-mobile.png"></a>
+        <div id="modal" class="overlay">
+          <a class="cancel" href="http://127.0.0.1:5500/code/index.html#"> </a>
```

Comment:
You anchor is empty, so there is no way of clicking it, we should add some text inside.
Moreover, `href` is not correct - I think you wanted write `href="index.html"`